<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DBBeagle</string>
  </property>
  <property name="windowIcon">
   <iconset resource="dbbeagle.qrc">
    <normaloff>:/icons/cs_beagle.png</normaloff>:/icons/cs_beagle.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>8</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="searchTab">
       <attribute name="title">
        <string>Search</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="searchControlsLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Search in tables:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="tablesLineEdit">
            <property name="text">
             <string>*</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="valueLineEdit"/>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="searchPushButton">
            <property name="text">
             <string comment="Perform search">Search</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Value:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="excludeTablesCheckBox">
            <property name="text">
             <string>Excluding</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="searchResultsTableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="sqlBrowserTab">
       <attribute name="title">
        <string>SQL Browser</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="sqlControlsLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Query:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="queryLineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="executeQueryPushButton">
            <property name="text">
             <string>Execute</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="queryTableView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="connectToDatasourceAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="searchInFoundAction"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonFollowStyle</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="connectToDatasourceAction"/>
   <addaction name="separator"/>
   <addaction name="searchInFoundAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="connectToDatasourceAction">
   <property name="icon">
    <iconset resource="dbbeagle.qrc">
     <normaloff>:/icons/svn-update.png</normaloff>:/icons/svn-update.png</iconset>
   </property>
   <property name="text">
    <string>Connect to a &amp;datasource...</string>
   </property>
   <property name="iconText">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Make new or switch existing connection to a datasource</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="icon">
    <iconset resource="dbbeagle.qrc">
     <normaloff>:/icons/application-exit.png</normaloff>:/icons/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="toolTip">
    <string>Exit the application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="searchInFoundAction">
   <property name="icon">
    <iconset resource="dbbeagle.qrc">
     <normaloff>:/icons/arrow-up-double.png</normaloff>:/icons/arrow-up-double.png</iconset>
   </property>
   <property name="text">
    <string>Search in the found tables</string>
   </property>
   <property name="iconText">
    <string>In found</string>
   </property>
   <property name="toolTip">
    <string>Use found table names as search criteria</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="dbbeagle.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>tablesLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>searchPushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>731</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valueLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>searchPushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>786</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>queryLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>executeQueryPushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>786</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
